### PR TITLE
App user sync

### PR DESCRIPTION
**Pull request by lemieux**
Messages are now sync between clients, thanks to the new Faye channel !

I also added sorting before rendering the messages as a goody.

This can't be merged until the backend is shipped with the new faye channel.

@alavers @mspensieri @dannytranlx @Mario54

### CODE REVIEW

File: src/js/components/conversation.jsx [hunk 1]
```diff
@@ -61,13 +61,26 @@ export class ConversationComponent extends Component {
...
 
     render() {
-        const messages = this.props.conversation.messages.map((message, index) => {
-            return <MessageComponent key={ index }
-                                     accentColor={ this.props.settings.accentColor }
-                                     linkColor={ this.props.settings.linkColor }
-                                     onLoad={ this.scrollToBottom }
-                                     {...message} />;
-        });
+        const messages = this.props.conversation.messages
```
Comment (mspensieri) [inline src/js/components/conversation.jsx]: Wouldn't it be better to sort these in the store?

Comment (lemieux) [inline src/js/components/conversation.jsx]: yup definitely going in the store

File: src/js/reducers/conversation-reducer.js [hunk 2]
```diff
@@ -6,16 +6,43 @@ const INITIAL_STATE = {
     unreadCount: 0
 };
 
+const sortMessage = (messages) => messages.sort((a, b) => {
```
Comment (alavers) [inline src/js/reducers/conversation-reducer.js]: nit, `sortMessages`

Comment (lemieux) [inline src/js/reducers/conversation-reducer.js]: err, went too fast on that one, nice catch :)

Comment (lemieux) [inline src/js/reducers/conversation-reducer.js]: fixed

**Comment by mspensieri** [comment]
:+1:

**Comment by Mario54** [comment]
👍

**Comment by alavers** [comment]
:+1:

**Pull request merged**